data/campaigns/An_Orcish_Incursion/scenarios/04_Valley_of_Trolls.cfg

```diff
@@ -78,17 +78,11 @@
         facing=ne
 
         [ai]
-            aggression=0.8
-            turns="4,5,6,10,11,12,16,17,18,22,23,24"
-            grouping=offensive
-            {ATTACK_DEPTH 1 3 5}
-        [/ai]
-        [ai]
-            aggression=-0.1
-            simple_targeting=yes
-            turns="1,2,3,7,8,9,13,14,15,19,20,21"
-            grouping=none
             {ATTACK_DEPTH 1 3 5}
+            {AI_SIMPLE_DAY_ASPECT aggression -0.1}
+            {AI_SIMPLE_DAY_ASPECT simple_targeting yes}
+            {AI_SIMPLE_DAY_ASPECT grouping no}
+            {AI_SIMPLE_NIGHT_ASPECT aggression 0.8}
```

Comment:
While it doesn't really matter one way or another, I actually prefer the older-style syntax for this, using two `[ai]` tags with different `time_of_day` values.